### ISSUE TITLE
Deprecate CMake and move to Meson

### ISSUE THREAD

**Issue by szszszsz** [Nitrokey/libnitrokey#181]
There is a proposition to use Meson as a main build system instead of CMake.
Let's discuss it here.

Source: https://github.com/Nitrokey/libnitrokey/pull/165#issuecomment-567577747

**Comment by robinkrahl** [comment]
What is the advantage of meson over CMake?

Personally, I don’t have a strong opinion on this issue, but I think it would be good to choose one and drop the other (unless there are reasons to keep both that I am not aware of).

**Comment by szszszsz** [comment]
I agree. Only one can be kept up-to-date and tested each release, and the other on the best effort.

It's hard to tell now, since the CMake has improved much since the moment the Meson was created (which was one of the reasons AFAIR). We do not have any technical problems with the former right now as far as I am concerned. Articles on the web are mentioning both as equal to
